src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -77,7 +77,11 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
                 commit({type: ModelCalibrateMutation.CalibrateResultFetched, payload: data});
                 commit({type: ModelCalibrateMutation.WarningsFetched, payload: data.warnings});
 
-                selectFilterDefaults(data, commit, PlottingSelectionsMutations.updateBarchartSelections)
+                const barChartPlotSelections = context.rootState.plottingSelections.barchart
+                // barchart reverts to default selections if selection does not exist in the state.
+                if (!barChartPlotSelections) {
```

Comment:
I don't think we should include this conditional  (though I understand why you added it!) - we shouldn't retain user plotting selections when the model has been recalibrated because those selections may no longer be valid for the new results. The point of this ticket was to retain user selections for the comparison plot when the user changes tab or step, not when there are new results